### DISCUSSION TITLE
能否在官方使用文档中说明defaultConfig的作用

### DISCUSSION THREAD

**Discussion by Luo-1Hao** [wangeditor-team/wangEditor#3858] [Q&A] [answered]
Whether the role of the property defaultConfig can be explained in the official documentation?

**Comment by wangfupeng1988** [comment, marked as answer]
看这里 https://www.wangeditor.com/v5/guide/for-frame.html ，如下图：

![image](https://user-images.githubusercontent.com/9583120/150747127-4b7c2fe8-27ba-4b1f-bda9-a4f90184f689.png)
